apps/swpcfg/README.md

```diff
@@ -0,0 +1,20 @@
+# Swipe Settings
+
+A service for configuring the Bangle.js scroll and swipe directions
+
+* **Scroll Direction** switch between natural (down motion scrolls down) and reverse (down motion scrolls up) vertical scrolling
```

Comment:
Isn't natural/reverse scrolling the other way around? Just so we get the terminology right.

One reference:
https://blog.logrocket.com/ux-design-natural-vs-reverse-scrolling/attachment/scroll-direction-natural-vs-reversed/

Reply:
I would name them "default" and "reverse"

Reply:
Good idea I think, since the default behavior is different across apps 👍